docs/docs/reference/other-new-features/named-typeargs.md

```diff
@@ -5,7 +5,7 @@ title: "Named Type Arguments"
 
 **Note:** This feature is implemented in Dotty, but is not expected to be part of Scala 3.0.
 
-Type arguments of methods can now be named, as well as by position. Example:
+Type arguments of methods can now be specified by name next to by position. Example:
```

Comment:
I don't think _next to_ is correct here. "can now be specified by name as well as by position"?